### PR TITLE
Increase `Enter config` event debounce timeout

### PR DESCRIPTION
**Pull request by a-chabin**

### CODE REVIEW

File: client/view/Form/Form.js [hunk 1]
```diff
@@ -171,19 +171,27 @@ form.addEventListener('submit', e => {
...
+  submitFormDebounced()
+})
 
 if (getUrlParams().config) {
   trackEvent('Open config', { props: getUrlParams() })
   submitFormWithUrlParams()
 }
 
+textarea.addEventListener('input', () => {
+  submitFormDebounced()
```
Comment (ai) [inline client/view/Form/Form.js:192]: Duplicate call of `submitFormDebounced`

File: client/lib/analytics.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import Plausible from 'plausible-tracker'
 
 let { trackEvent, trackPageview } = Plausible({
-  domain: 'browsersl.ist'
+  domain: 'test.test',
+  trackLocalhost: true
```
Comment (ai) [inline client/lib/analytics.js:5]: 😏

File: client/view/Form/Form.js (hunk 1 above)
Comment (a-chabin) [reply client/view/Form/Form.js]: Fixed

File: client/lib/analytics.js (hunk 2 above)
Comment (a-chabin) [reply client/lib/analytics.js]: 🤩

**Pull request merged**